core/src/main/web/app/mainapp/components/notebook/codecell-directive.js

```diff
@@ -420,6 +426,32 @@
           }
         });
 
+        scope.evaluateSelection = function(cm) {
+          var evalCode;
+          var currentLine;
+          if(cm.somethingSelected()){
```

Comment:
official style is to have a space between "if" and "(" and between ")" and "{".
also should use "} else {" etc.
you don't have to fix this PR, it's not that important, but please do it this way going forward.